mod_auth/forms.py

```diff
@@ -33,12 +38,25 @@ def _email_not_in_use(form, field):
 
 
 def role_id_is_valid(form, field):
+	'''
+	Checks for validity of User's Role
+	'''
     role = Role.query.filter(Role.id == field.data).first()
     if role is None:
         raise ValidationError('Role id is invalid')
 
 
 class LoginForm(Form):
+	'''
+	Form for Login
+
+	attr email: user's email
```

Comment:
This documentation doesn't provide any useful information (attr) to the user, and for an IDE it's not needed either.